src/api/AuthServer/AuthServer/ApplicationUser.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace AuthServer
+{
+    public class ApplicationUser : IdentityUser
+    {
+        public string Name { get; set; }
+        public string Surname { get; set; }
+
```

Comment:
Empty spaces between property declarations. Remove empty space at the end of the class